idaes/models/unit_models/separator.py

```diff
@@ -209,6 +210,22 @@ class SeparatorData(UnitModelBlockData):
 **MaterialBalanceType.total** - use total material balance.}""",
         ),
     )
+    CONFIG.declare(
+        "momentum_balance_type",
+        ConfigValue(
+            default=MomentumBalanceType.pressureTotal,
+            domain=In(MomentumBalanceType),
+            description="Momentum balance construction flag",
+            doc="""Indicates what type of momentum balance should be constructed,
+    **default** - MomentumBalanceType.pressureTotal.
+    **Valid values:** {
+    **MomentumBalanceType.none** - exclude momentum balances,
+    **MomentumBalanceType.pressureTotal** - single pressure balance for material,
```

Comment:
This should be updated to reflect what the Separator does - i.e. if momentum balance is none then no balance is written, otherwise pressure is equal in all outlets.

Reply:
I added a comment (pressure in all outlets is equivalent) for each option except `none`.